TID: [-1234] [2020-06-03 11:09:26,111]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 11:09:26,114]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 11:09:26,115]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 11:09:26,117]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 11:18:43,294] ERROR {org.wso2.carbon.dataservices.core.DBDeployer} - The SORTEO.dbs service, which is not valid, caused {1} DS Code: UNKNOWN_ERROR
Source Data Service:-
Name: SORTEO
Location: \SORTEO.dbs
Description: N/A
Default Namespace: http://ws.wso2.org/dataservice
Nested Exception:-
java.lang.StringIndexOutOfBoundsException: String index out of range: -1

	at org.wso2.carbon.dataservices.core.DataServiceFactory.createDataService(DataServiceFactory.java:208)
	at org.wso2.carbon.dataservices.core.DBDeployer.createDBService(DBDeployer.java:811)
	at org.wso2.carbon.dataservices.core.DBDeployer.processService(DBDeployer.java:1169)
	at org.wso2.carbon.dataservices.core.DBDeployer.deploy(DBDeployer.java:202)
	at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
	at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:807)
	at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
	at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
	at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
	at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:93)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.run(CarbonDeploymentSchedulerTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: -1
	at java.lang.String.substring(String.java:1967)
	at org.wso2.carbon.dataservices.core.description.query.SQLQuery.sqlQueryType(SQLQuery.java:2528)
	at org.wso2.carbon.dataservices.core.description.query.SQLQuery.setSqlQueryType(SQLQuery.java:2523)
	at org.wso2.carbon.dataservices.core.description.query.SQLQuery.init(SQLQuery.java:200)
	at org.wso2.carbon.dataservices.core.description.query.SQLQuery.<init>(SQLQuery.java:185)
	at org.wso2.carbon.dataservices.core.description.query.QueryFactory.createSQLQuery(QueryFactory.java:444)
	at org.wso2.carbon.dataservices.core.description.query.QueryFactory.createQuery(QueryFactory.java:95)
	at org.wso2.carbon.dataservices.core.DataServiceFactory.createDataService(DataServiceFactory.java:173)
	... 20 more

TID: [-1234] [2020-06-03 11:49:59,355] ERROR {org.apache.synapse.mediators.bsf.ScriptMediator} - The script engine returned an error executing the inlined js script function mediate com.sun.phobos.script.util.ExtendedScriptException: org.mozilla.javascript.EcmaError: ReferenceError: "newPayload" is not defined. (<Unknown Source>#1) in <Unknown Source> at line number 1
	at com.sun.phobos.script.javascript.RhinoCompiledScript.eval(RhinoCompiledScript.java:68)
	at javax.script.CompiledScript.eval(CompiledScript.java:92)
	at org.apache.synapse.mediators.bsf.ScriptMediator.mediateForInlineScript(ScriptMediator.java:395)
	at org.apache.synapse.mediators.bsf.ScriptMediator.invokeScript(ScriptMediator.java:290)
	at org.apache.synapse.mediators.bsf.ScriptMediator.mediate(ScriptMediator.java:258)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.rest.Resource.process(Resource.java:331)
	at org.apache.synapse.rest.API.process(API.java:441)
	at org.apache.synapse.rest.RESTRequestHandler.apiProcess(RESTRequestHandler.java:135)
	at org.apache.synapse.rest.RESTRequestHandler.dispatchToAPI(RESTRequestHandler.java:113)
	at org.apache.synapse.rest.RESTRequestHandler.process(RESTRequestHandler.java:71)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:327)
	at org.apache.synapse.core.axis2.SynapseMessageReceiver.receive(SynapseMessageReceiver.java:98)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ServerWorker.processNonEntityEnclosingRESTHandler(ServerWorker.java:368)
	at org.apache.synapse.transport.passthru.ServerWorker.run(ServerWorker.java:189)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.mozilla.javascript.EcmaError: ReferenceError: "newPayload" is not defined. (<Unknown Source>#1)
	at org.mozilla.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3687)
	at org.mozilla.javascript.ScriptRuntime.constructError(ScriptRuntime.java:3665)
	at org.mozilla.javascript.ScriptRuntime.notFoundError(ScriptRuntime.java:3750)
	at org.mozilla.javascript.ScriptRuntime.name(ScriptRuntime.java:1728)
	at org.mozilla.javascript.gen._Unknown_Source__425._c_script_0(<Unknown Source>:1)
	at org.mozilla.javascript.gen._Unknown_Source__425.call(<Unknown Source>)
	at org.mozilla.javascript.ContextFactory.doTopCall(ContextFactory.java:394)
	at org.mozilla.javascript.ScriptRuntime.doTopCall(ScriptRuntime.java:3091)
	at org.mozilla.javascript.gen._Unknown_Source__425.call(<Unknown Source>)
	at org.mozilla.javascript.gen._Unknown_Source__425.exec(<Unknown Source>)
	at com.sun.phobos.script.javascript.RhinoCompiledScript.eval(RhinoCompiledScript.java:55)
	... 21 more

TID: [-1234] [2020-06-03 16:56:53,743] ERROR {org.apache.synapse.mediators.db.DBLookupMediator} - SQL Exception occurred while executing statement : CALL sp_ProcesarInsertaPeticion (?) against DataSource : Mysql com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:990)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:342)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2197)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2230)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2025)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:778)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:386)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at org.apache.tomcat.jdbc.pool.PooledConnection.connectUsingDriver(PooledConnection.java:319)
	at org.apache.tomcat.jdbc.pool.PooledConnection.connect(PooledConnection.java:212)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.createConnection(ConnectionPool.java:739)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:671)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.getConnection(ConnectionPool.java:198)
	at org.apache.tomcat.jdbc.pool.DataSourceProxy.getConnection(DataSourceProxy.java:132)
	at org.apache.synapse.mediators.db.DBLookupMediator.processStatement(DBLookupMediator.java:58)
	at org.apache.synapse.mediators.db.AbstractDBMediator.mediate(AbstractDBMediator.java:243)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.rest.Resource.process(Resource.java:331)
	at org.apache.synapse.rest.API.process(API.java:441)
	at org.apache.synapse.rest.RESTRequestHandler.apiProcess(RESTRequestHandler.java:135)
	at org.apache.synapse.rest.RESTRequestHandler.dispatchToAPI(RESTRequestHandler.java:113)
	at org.apache.synapse.rest.RESTRequestHandler.process(RESTRequestHandler.java:71)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:327)
	at org.apache.synapse.core.axis2.SynapseMessageReceiver.receive(SynapseMessageReceiver.java:98)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ServerWorker.processNonEntityEnclosingRESTHandler(ServerWorker.java:368)
	at org.apache.synapse.transport.passthru.ServerWorker.run(ServerWorker.java:189)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:301)
	... 37 more

TID: [-1234] [2020-06-03 16:58:16,315] ERROR {org.wso2.carbon.ndatasource.core.DataSourceRepository} - Error establishing data source connection: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server. org.wso2.carbon.ndatasource.common.DataSourceException: Error establishing data source connection: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader.testDataSourceConnection(RDBMSDataSourceReader.java:95)
	at org.wso2.carbon.ndatasource.core.DataSourceRepository.testDataSourceConnection(DataSourceRepository.java:513)
	at org.wso2.carbon.ndatasource.core.services.NDataSourceAdminService.testDataSourceConnection(NDataSourceAdminService.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.axis2.rpc.receivers.RPCUtil.invokeServiceClass(RPCUtil.java:212)
	at org.apache.axis2.rpc.receivers.RPCMessageReceiver.invokeBusinessLogic(RPCMessageReceiver.java:117)
	at org.apache.axis2.receivers.AbstractInOutMessageReceiver.invokeBusinessLogic(AbstractInOutMessageReceiver.java:40)
	at org.apache.axis2.receivers.AbstractMessageReceiver.receive(AbstractMessageReceiver.java:110)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.axis2.transport.local.LocalTransportReceiver.processMessage(LocalTransportReceiver.java:170)
	at org.apache.axis2.transport.local.LocalTransportReceiver.processMessage(LocalTransportReceiver.java:82)
	at org.wso2.carbon.core.transports.local.CarbonLocalTransportSender.finalizeSendWithToAddress(CarbonLocalTransportSender.java:45)
	at org.apache.axis2.transport.local.LocalTransportSender.invoke(LocalTransportSender.java:77)
	at org.apache.axis2.engine.AxisEngine.send(AxisEngine.java:442)
	at org.apache.axis2.description.OutInAxisOperationClient.send(OutInAxisOperation.java:442)
	at org.apache.axis2.description.OutInAxisOperationClient.executeImpl(OutInAxisOperation.java:228)
	at org.apache.axis2.client.OperationClient.execute(OperationClient.java:149)
	at org.wso2.carbon.ndatasource.ui.stub.NDataSourceAdminStub.testDataSourceConnection(NDataSourceAdminStub.java:2520)
	at org.wso2.carbon.ndatasource.ui.NDataSourceAdminServiceClient.testDataSourceConnection(NDataSourceAdminServiceClient.java:88)
	at org.apache.jsp.ndatasource.validateconnection_002dajaxprocessor_jsp._jspService(validateconnection_002dajaxprocessor_jsp.java:139)
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:70)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:476)
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:385)
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:329)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.wso2.carbon.ui.JspServlet.service(JspServlet.java:207)
	at org.wso2.carbon.ui.TilesJspServlet.service(TilesJspServlet.java:80)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.eclipse.equinox.http.helper.ContextPathServletAdaptor.service(ContextPathServletAdaptor.java:37)
	at org.eclipse.equinox.http.servlet.internal.ServletRegistration.service(ServletRegistration.java:61)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.processAlias(ProxyServlet.java:128)
	at org.eclipse.equinox.http.servlet.internal.ProxyServlet.service(ProxyServlet.java:68)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.wso2.carbon.tomcat.ext.servlet.DelegationServlet.service(DelegationServlet.java:68)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:53)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.wso2.carbon.ui.filters.cache.URLBasedCachePreventionFilter.doFilter(URLBasedCachePreventionFilter.java:57)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.owasp.csrfguard.CsrfGuardFilter.doFilter(CsrfGuardFilter.java:88)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.wso2.carbon.ui.filters.CSRFPreventionFilter.doFilter(CSRFPreventionFilter.java:88)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.wso2.carbon.ui.filters.CRLFPreventionFilter.doFilter(CRLFPreventionFilter.java:61)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.wso2.carbon.tomcat.ext.filter.CharacterSetFilter.doFilter(CharacterSetFilter.java:65)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.filters.HttpHeaderSecurityFilter.doFilter(HttpHeaderSecurityFilter.java:126)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:202)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:490)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:139)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:92)
	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.continueInvocation(CompositeValve.java:99)
	at org.wso2.carbon.tomcat.ext.valves.TomcatValveContainer.invokeValves(TomcatValveContainer.java:49)
	at org.wso2.carbon.tomcat.ext.valves.CompositeValve.invoke(CompositeValve.java:62)
	at org.wso2.carbon.tomcat.ext.valves.CarbonStuckThreadDetectionValve.invoke(CarbonStuckThreadDetectionValve.java:146)
	at org.apache.catalina.valves.AbstractAccessLogValve.invoke(AbstractAccessLogValve.java:678)
	at org.wso2.carbon.tomcat.ext.valves.CarbonContextCreatorValve.invoke(CarbonContextCreatorValve.java:57)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:408)
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:853)
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1587)
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
	at java.lang.Thread.run(Thread.java:748)
Caused by: com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:990)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:342)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2197)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2230)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2025)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:778)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:386)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at java.sql.DriverManager.getConnection(DriverManager.java:664)
	at java.sql.DriverManager.getConnection(DriverManager.java:247)
	at org.wso2.carbon.ndatasource.rdbms.RDBMSDataSourceReader.testDataSourceConnection(RDBMSDataSourceReader.java:80)
	... 82 more
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:301)
	... 97 more

TID: [-1234] [2020-06-03 17:00:23,856] ERROR {org.apache.synapse.mediators.db.DBLookupMediator} - SQL Exception occurred while executing statement : CALL sp_ProcesarInsertaPeticion (?) against DataSource : Mysql com.mysql.jdbc.exceptions.jdbc4.CommunicationsException: Communications link failure

The last packet sent successfully to the server was 0 milliseconds ago. The driver has not received any packets from the server.
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.SQLError.createCommunicationsException(SQLError.java:990)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:342)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2197)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2230)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2025)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:778)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:425)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:386)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:330)
	at org.apache.tomcat.jdbc.pool.PooledConnection.connectUsingDriver(PooledConnection.java:319)
	at org.apache.tomcat.jdbc.pool.PooledConnection.connect(PooledConnection.java:212)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.createConnection(ConnectionPool.java:739)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.borrowConnection(ConnectionPool.java:671)
	at org.apache.tomcat.jdbc.pool.ConnectionPool.getConnection(ConnectionPool.java:198)
	at org.apache.tomcat.jdbc.pool.DataSourceProxy.getConnection(DataSourceProxy.java:132)
	at org.apache.synapse.mediators.db.DBLookupMediator.processStatement(DBLookupMediator.java:58)
	at org.apache.synapse.mediators.db.AbstractDBMediator.mediate(AbstractDBMediator.java:243)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.rest.Resource.process(Resource.java:331)
	at org.apache.synapse.rest.API.process(API.java:441)
	at org.apache.synapse.rest.RESTRequestHandler.apiProcess(RESTRequestHandler.java:135)
	at org.apache.synapse.rest.RESTRequestHandler.dispatchToAPI(RESTRequestHandler.java:113)
	at org.apache.synapse.rest.RESTRequestHandler.process(RESTRequestHandler.java:71)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:327)
	at org.apache.synapse.core.axis2.SynapseMessageReceiver.receive(SynapseMessageReceiver.java:98)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ServerWorker.processNonEntityEnclosingRESTHandler(ServerWorker.java:368)
	at org.apache.synapse.transport.passthru.ServerWorker.run(ServerWorker.java:189)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection timed out: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(DualStackPlainSocketImpl.java:79)
	at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)
	at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)
	at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:172)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)
	at java.net.Socket.connect(Socket.java:589)
	at com.mysql.jdbc.StandardSocketFactory.connect(StandardSocketFactory.java:211)
	at com.mysql.jdbc.MysqlIO.<init>(MysqlIO.java:301)
	... 37 more

TID: [-1234] [2020-06-03 17:02:33,273]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 17:02:33,274]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 17:02:33,276]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 17:02:33,280]  WARN {org.apache.tomcat.util.net.SSLUtilBase} - jsseUtil.trustedCertNotValid
TID: [-1234] [2020-06-03 17:19:19,909] ERROR {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} - Update of the Synapse Artifact from file : C:\Program Files\WSO2\Enterprise Integrator\6.6.0\repository\deployment\server\synapse-configs\default\api\wsProcesarFovissste.xml : Failed! org.apache.synapse.SynapseException: Exception initializing inline script
	at org.apache.synapse.mediators.bsf.ScriptMediator.initInlineScript(ScriptMediator.java:462)
	at org.apache.synapse.mediators.bsf.ScriptMediator.<init>(ScriptMediator.java:193)
	at org.apache.synapse.mediators.bsf.ScriptMediatorFactory.createSpecificMediator(ScriptMediatorFactory.java:105)
	at org.apache.synapse.config.xml.AbstractMediatorFactory.createMediator(AbstractMediatorFactory.java:96)
	at org.apache.synapse.config.xml.MediatorFactoryFinder.getMediator(MediatorFactoryFinder.java:224)
	at org.apache.synapse.config.xml.AbstractListMediatorFactory.addChildren(AbstractListMediatorFactory.java:46)
	at org.apache.synapse.config.xml.SequenceMediatorFactory.createAnonymousSequence(SequenceMediatorFactory.java:70)
	at org.apache.synapse.config.xml.rest.ResourceFactory.configureSequences(ResourceFactory.java:119)
	at org.apache.synapse.config.xml.rest.ResourceFactory.createResource(ResourceFactory.java:50)
	at org.apache.synapse.config.xml.rest.APIFactory.createAPI(APIFactory.java:94)
	at org.apache.synapse.deployers.APIDeployer.updateSynapseArtifact(APIDeployer.java:81)
	at org.wso2.carbon.rest.api.ApiDeployer.updateSynapseArtifact(ApiDeployer.java:41)
	at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:165)
	at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
	at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:807)
	at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
	at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
	at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
	at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:93)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.run(CarbonDeploymentSchedulerTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: javax.script.ScriptException: org.mozilla.javascript.EvaluatorException: syntax error (<Unknown Source>#1)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:341)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:323)
	at org.apache.synapse.mediators.bsf.ScriptMediator.initInlineScript(ScriptMediator.java:451)
	... 29 more
Caused by: org.mozilla.javascript.EvaluatorException: syntax error (<Unknown Source>#1)
	at org.mozilla.javascript.DefaultErrorReporter.runtimeError(DefaultErrorReporter.java:77)
	at org.mozilla.javascript.DefaultErrorReporter.error(DefaultErrorReporter.java:64)
	at org.mozilla.javascript.Parser.addError(Parser.java:188)
	at org.mozilla.javascript.Parser.addError(Parser.java:166)
	at org.mozilla.javascript.Parser.reportError(Parser.java:223)
	at org.mozilla.javascript.Parser.reportError(Parser.java:210)
	at org.mozilla.javascript.Parser.reportError(Parser.java:203)
	at org.mozilla.javascript.Parser.primaryExpr(Parser.java:2825)
	at org.mozilla.javascript.Parser.memberExpr(Parser.java:2406)
	at org.mozilla.javascript.Parser.unaryExpr(Parser.java:2304)
	at org.mozilla.javascript.Parser.mulExpr(Parser.java:2237)
	at org.mozilla.javascript.Parser.addExpr(Parser.java:2213)
	at org.mozilla.javascript.Parser.shiftExpr(Parser.java:2194)
	at org.mozilla.javascript.Parser.relExpr(Parser.java:2169)
	at org.mozilla.javascript.Parser.eqExpr(Parser.java:2141)
	at org.mozilla.javascript.Parser.bitAndExpr(Parser.java:2130)
	at org.mozilla.javascript.Parser.bitXorExpr(Parser.java:2119)
	at org.mozilla.javascript.Parser.bitOrExpr(Parser.java:2108)
	at org.mozilla.javascript.Parser.andExpr(Parser.java:2097)
	at org.mozilla.javascript.Parser.orExpr(Parser.java:2086)
	at org.mozilla.javascript.Parser.condExpr(Parser.java:2062)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:2033)
	at org.mozilla.javascript.Parser.argumentList(Parser.java:2373)
	at org.mozilla.javascript.Parser.memberExprTail(Parser.java:2523)
	at org.mozilla.javascript.Parser.memberExpr(Parser.java:2442)
	at org.mozilla.javascript.Parser.unaryExpr(Parser.java:2304)
	at org.mozilla.javascript.Parser.mulExpr(Parser.java:2229)
	at org.mozilla.javascript.Parser.addExpr(Parser.java:2213)
	at org.mozilla.javascript.Parser.shiftExpr(Parser.java:2194)
	at org.mozilla.javascript.Parser.relExpr(Parser.java:2169)
	at org.mozilla.javascript.Parser.eqExpr(Parser.java:2141)
	at org.mozilla.javascript.Parser.bitAndExpr(Parser.java:2130)
	at org.mozilla.javascript.Parser.bitXorExpr(Parser.java:2119)
	at org.mozilla.javascript.Parser.bitOrExpr(Parser.java:2108)
	at org.mozilla.javascript.Parser.andExpr(Parser.java:2097)
	at org.mozilla.javascript.Parser.orExpr(Parser.java:2086)
	at org.mozilla.javascript.Parser.condExpr(Parser.java:2062)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:2033)
	at org.mozilla.javascript.Parser.expr(Parser.java:2012)
	at org.mozilla.javascript.Parser.nameOrLabel(Parser.java:1765)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:1032)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.statements(Parser.java:856)
	at org.mozilla.javascript.Parser.block(Parser.java:1688)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:1010)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.ifStatement(Parser.java:1085)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:947)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.parse(Parser.java:540)
	at org.mozilla.javascript.Parser.parse(Parser.java:502)
	at org.mozilla.javascript.Context.compileImpl(Context.java:2350)
	at org.mozilla.javascript.Context.compileReader(Context.java:1296)
	at org.mozilla.javascript.Context.compileReader(Context.java:1268)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:337)
	... 31 more

TID: [-1234] [2020-06-03 17:19:20,360] ERROR {org.apache.synapse.deployers.AbstractSynapseArtifactDeployer} - Deployment of synapse artifact failed. Error reading C:\Program Files\WSO2\Enterprise Integrator\6.6.0\repository\deployment\server\synapse-configs\default\api\wsProcesarFovissste.xml : Exception initializing inline script org.apache.axis2.deployment.DeploymentException: Exception initializing inline script
	at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:178)
	at org.apache.axis2.deployment.repository.util.DeploymentFileData.deploy(DeploymentFileData.java:136)
	at org.apache.axis2.deployment.DeploymentEngine.doDeploy(DeploymentEngine.java:807)
	at org.apache.axis2.deployment.repository.util.WSInfoList.update(WSInfoList.java:144)
	at org.apache.axis2.deployment.RepositoryListener.update(RepositoryListener.java:377)
	at org.apache.axis2.deployment.RepositoryListener.checkServices(RepositoryListener.java:254)
	at org.apache.axis2.deployment.RepositoryListener.startListener(RepositoryListener.java:371)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.checkRepository(SchedulerTask.java:59)
	at org.apache.axis2.deployment.scheduler.SchedulerTask.run(SchedulerTask.java:67)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.runAxisDeployment(CarbonDeploymentSchedulerTask.java:93)
	at org.wso2.carbon.core.deployment.CarbonDeploymentSchedulerTask.run(CarbonDeploymentSchedulerTask.java:138)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.apache.synapse.SynapseException: Exception initializing inline script
	at org.apache.synapse.mediators.bsf.ScriptMediator.initInlineScript(ScriptMediator.java:462)
	at org.apache.synapse.mediators.bsf.ScriptMediator.<init>(ScriptMediator.java:193)
	at org.apache.synapse.mediators.bsf.ScriptMediatorFactory.createSpecificMediator(ScriptMediatorFactory.java:105)
	at org.apache.synapse.config.xml.AbstractMediatorFactory.createMediator(AbstractMediatorFactory.java:96)
	at org.apache.synapse.config.xml.MediatorFactoryFinder.getMediator(MediatorFactoryFinder.java:224)
	at org.apache.synapse.config.xml.AbstractListMediatorFactory.addChildren(AbstractListMediatorFactory.java:46)
	at org.apache.synapse.config.xml.SequenceMediatorFactory.createAnonymousSequence(SequenceMediatorFactory.java:70)
	at org.apache.synapse.config.xml.rest.ResourceFactory.configureSequences(ResourceFactory.java:119)
	at org.apache.synapse.config.xml.rest.ResourceFactory.createResource(ResourceFactory.java:50)
	at org.apache.synapse.config.xml.rest.APIFactory.createAPI(APIFactory.java:94)
	at org.apache.synapse.deployers.APIDeployer.updateSynapseArtifact(APIDeployer.java:81)
	at org.wso2.carbon.rest.api.ApiDeployer.updateSynapseArtifact(ApiDeployer.java:41)
	at org.apache.synapse.deployers.AbstractSynapseArtifactDeployer.deploy(AbstractSynapseArtifactDeployer.java:165)
	... 17 more
Caused by: javax.script.ScriptException: org.mozilla.javascript.EvaluatorException: syntax error (<Unknown Source>#1)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:341)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:323)
	at org.apache.synapse.mediators.bsf.ScriptMediator.initInlineScript(ScriptMediator.java:451)
	... 29 more
Caused by: org.mozilla.javascript.EvaluatorException: syntax error (<Unknown Source>#1)
	at org.mozilla.javascript.DefaultErrorReporter.runtimeError(DefaultErrorReporter.java:77)
	at org.mozilla.javascript.DefaultErrorReporter.error(DefaultErrorReporter.java:64)
	at org.mozilla.javascript.Parser.addError(Parser.java:188)
	at org.mozilla.javascript.Parser.addError(Parser.java:166)
	at org.mozilla.javascript.Parser.reportError(Parser.java:223)
	at org.mozilla.javascript.Parser.reportError(Parser.java:210)
	at org.mozilla.javascript.Parser.reportError(Parser.java:203)
	at org.mozilla.javascript.Parser.primaryExpr(Parser.java:2825)
	at org.mozilla.javascript.Parser.memberExpr(Parser.java:2406)
	at org.mozilla.javascript.Parser.unaryExpr(Parser.java:2304)
	at org.mozilla.javascript.Parser.mulExpr(Parser.java:2237)
	at org.mozilla.javascript.Parser.addExpr(Parser.java:2213)
	at org.mozilla.javascript.Parser.shiftExpr(Parser.java:2194)
	at org.mozilla.javascript.Parser.relExpr(Parser.java:2169)
	at org.mozilla.javascript.Parser.eqExpr(Parser.java:2141)
	at org.mozilla.javascript.Parser.bitAndExpr(Parser.java:2130)
	at org.mozilla.javascript.Parser.bitXorExpr(Parser.java:2119)
	at org.mozilla.javascript.Parser.bitOrExpr(Parser.java:2108)
	at org.mozilla.javascript.Parser.andExpr(Parser.java:2097)
	at org.mozilla.javascript.Parser.orExpr(Parser.java:2086)
	at org.mozilla.javascript.Parser.condExpr(Parser.java:2062)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:2033)
	at org.mozilla.javascript.Parser.argumentList(Parser.java:2373)
	at org.mozilla.javascript.Parser.memberExprTail(Parser.java:2523)
	at org.mozilla.javascript.Parser.memberExpr(Parser.java:2442)
	at org.mozilla.javascript.Parser.unaryExpr(Parser.java:2304)
	at org.mozilla.javascript.Parser.mulExpr(Parser.java:2229)
	at org.mozilla.javascript.Parser.addExpr(Parser.java:2213)
	at org.mozilla.javascript.Parser.shiftExpr(Parser.java:2194)
	at org.mozilla.javascript.Parser.relExpr(Parser.java:2169)
	at org.mozilla.javascript.Parser.eqExpr(Parser.java:2141)
	at org.mozilla.javascript.Parser.bitAndExpr(Parser.java:2130)
	at org.mozilla.javascript.Parser.bitXorExpr(Parser.java:2119)
	at org.mozilla.javascript.Parser.bitOrExpr(Parser.java:2108)
	at org.mozilla.javascript.Parser.andExpr(Parser.java:2097)
	at org.mozilla.javascript.Parser.orExpr(Parser.java:2086)
	at org.mozilla.javascript.Parser.condExpr(Parser.java:2062)
	at org.mozilla.javascript.Parser.assignExpr(Parser.java:2033)
	at org.mozilla.javascript.Parser.expr(Parser.java:2012)
	at org.mozilla.javascript.Parser.nameOrLabel(Parser.java:1765)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:1032)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.statements(Parser.java:856)
	at org.mozilla.javascript.Parser.block(Parser.java:1688)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:1010)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.ifStatement(Parser.java:1085)
	at org.mozilla.javascript.Parser.statementHelper(Parser.java:947)
	at org.mozilla.javascript.Parser.statement(Parser.java:901)
	at org.mozilla.javascript.Parser.parse(Parser.java:540)
	at org.mozilla.javascript.Parser.parse(Parser.java:502)
	at org.mozilla.javascript.Context.compileImpl(Context.java:2350)
	at org.mozilla.javascript.Context.compileReader(Context.java:1296)
	at org.mozilla.javascript.Context.compileReader(Context.java:1268)
	at com.sun.phobos.script.javascript.RhinoScriptEngine.compile(RhinoScriptEngine.java:337)
	... 31 more

TID: [-1234] [2020-06-03 17:24:01,411] ERROR {org.apache.synapse.mediators.db.DBLookupMediator} - Error executing statement : CALL sp_Procesar_detalle (?, ?) against DataSource : Mysql java.lang.NumberFormatException: For input string: "<icp xmlns:ns2="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/" xmlns:ns3="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/consultarSaldoFovissste/" xmlns:ns4="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoCurpFovissste/" xmlns:ns5="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/desmarcarCreditoFovissste/" xmlns:ns6="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoIdProcesarFovissste/">6091930</icp>
<icp xmlns:ns2="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/" xmlns:ns3="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/consultarSaldoFovissste/" xmlns:ns4="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoCurpFovissste/" xmlns:ns5="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/desmarcarCreditoFovissste/" xmlns:ns6="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoIdProcesarFovissste/">6092372</icp>"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at org.apache.synapse.mediators.db.AbstractDBMediator.getPreparedStatement(AbstractDBMediator.java:438)
	at org.apache.synapse.mediators.db.DBLookupMediator.processStatement(DBLookupMediator.java:62)
	at org.apache.synapse.mediators.db.AbstractDBMediator.mediate(AbstractDBMediator.java:243)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.rest.Resource.process(Resource.java:331)
	at org.apache.synapse.rest.API.process(API.java:380)
	at org.apache.synapse.rest.RESTRequestHandler.apiProcess(RESTRequestHandler.java:135)
	at org.apache.synapse.rest.RESTRequestHandler.dispatchToAPI(RESTRequestHandler.java:113)
	at org.apache.synapse.rest.RESTRequestHandler.process(RESTRequestHandler.java:58)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:327)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.handleMessage(SynapseCallbackReceiver.java:578)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.receive(SynapseCallbackReceiver.java:195)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ClientWorker.run(ClientWorker.java:284)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

TID: [-1] [2020-06-03 17:27:00,894]  WARN {org.apache.synapse.transport.passthru.SourceHandler} - Connection time out after request is read: http-incoming-4 Socket Timeout : 180000 Remote Address : /127.0.0.1:59907
TID: [-1234] [2020-06-03 17:27:01,556] ERROR {org.apache.synapse.mediators.db.DBLookupMediator} - Error executing statement : CALL sp_Procesar_detalle (?, ?) against DataSource : Mysql java.lang.NumberFormatException: For input string: "<icp xmlns:ns2="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/" xmlns:ns3="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/consultarSaldoFovissste/" xmlns:ns4="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoCurpFovissste/" xmlns:ns5="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/desmarcarCreditoFovissste/" xmlns:ns6="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoIdProcesarFovissste/">6091930</icp>
<icp xmlns:ns2="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/" xmlns:ns3="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/consultarSaldoFovissste/" xmlns:ns4="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoCurpFovissste/" xmlns:ns5="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/desmarcarCreditoFovissste/" xmlns:ns6="http://www.procesar.com.mx/TraspasosInfonavitFovissste/Fovissste/SolicitudCredito/marcarCreditoIdProcesarFovissste/">6092372</icp>"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Integer.parseInt(Integer.java:580)
	at java.lang.Integer.parseInt(Integer.java:615)
	at org.apache.synapse.mediators.db.AbstractDBMediator.getPreparedStatement(AbstractDBMediator.java:438)
	at org.apache.synapse.mediators.db.DBLookupMediator.processStatement(DBLookupMediator.java:62)
	at org.apache.synapse.mediators.db.AbstractDBMediator.mediate(AbstractDBMediator.java:243)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:109)
	at org.apache.synapse.mediators.AbstractListMediator.mediate(AbstractListMediator.java:71)
	at org.apache.synapse.mediators.base.SequenceMediator.mediate(SequenceMediator.java:158)
	at org.apache.synapse.rest.Resource.process(Resource.java:331)
	at org.apache.synapse.rest.API.process(API.java:380)
	at org.apache.synapse.rest.RESTRequestHandler.apiProcess(RESTRequestHandler.java:135)
	at org.apache.synapse.rest.RESTRequestHandler.dispatchToAPI(RESTRequestHandler.java:113)
	at org.apache.synapse.rest.RESTRequestHandler.process(RESTRequestHandler.java:58)
	at org.apache.synapse.core.axis2.Axis2SynapseEnvironment.injectMessage(Axis2SynapseEnvironment.java:327)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.handleMessage(SynapseCallbackReceiver.java:578)
	at org.apache.synapse.core.axis2.SynapseCallbackReceiver.receive(SynapseCallbackReceiver.java:195)
	at org.apache.axis2.engine.AxisEngine.receive(AxisEngine.java:180)
	at org.apache.synapse.transport.passthru.ClientWorker.run(ClientWorker.java:284)
	at org.apache.axis2.transport.base.threads.NativeWorkerPool$1.run(NativeWorkerPool.java:172)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)

TID: [-1] [2020-06-03 17:30:01,050]  WARN {org.apache.synapse.transport.passthru.SourceHandler} - Connection time out after request is read: http-incoming-5 Socket Timeout : 180000 Remote Address : /127.0.0.1:60074
